test/e2e/next-image-forward-ref/index.test.ts

```diff
@@ -0,0 +1,33 @@
+import { createNext, FileRef } from 'e2e-utils'
+import { NextInstance } from 'test/lib/next-modes/base'
+import { waitFor } from 'next-test-utils'
+import path from 'path'
+import webdriver from 'next-webdriver'
+
+describe('next-image-forward-ref', () => {
+  let next: NextInstance
+
+  const appDir = path.join(__dirname, 'app')
+
+  beforeAll(async () => {
+    next = await createNext({
+      files: new FileRef(appDir),
+      dependencies: {
+        'framer-motion': '7.6.9',
+      },
+    })
+  })
+  afterAll(() => next.destroy())
+
+  it('allows framer-motion to animate opacity', async () => {
+    const browser = await webdriver(next.url, '/framer-motion')
+    expect(
+      Number(await browser.elementById('img').getComputedCss('opacity'))
+    ).toBeCloseTo(1)
```

Comment:
```suggestion
    ).toBe(1)
```

Is there a scenario where this wouldn't be 1?

Reply:
It should be 1. But IDK what framer does internally and `===` with floats is dangerous.
`toBeClose` has tolerance of `0.01` so it still asserts that it's animating.
But `toBe` will probably work too, so I'll leave that up to you.

Reply:
I know that JavaScript generally does a good job at rounding to whole numbers so getting `0.999999978656` instead of `1` is not that common but I think that it's still a good practice.